### ISSUE TITLE
Accessibility - Links Fail Contrast Compliance

### ISSUE THREAD

**Issue by vkoves** [chihacknight/decarbonize-my-state#77]
Found using [Axe DevTools](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles) that there are three real contrast issues, all related to links. We should darken these and then re-check with AxeDevTools or [contrast-ratio](https://contrast-ratio.com/).

The issues are:

| Header links | Footer link | Links in State Details Action Panel |
| --- | --- | --- | 
| ![Screenshot from 2022-05-10 19-14-25](https://user-images.githubusercontent.com/3187531/167744883-fdd2399d-313a-472c-aec9-17994e4098b1.png) | ![Screenshot from 2022-05-10 19-14-42](https://user-images.githubusercontent.com/3187531/167744882-bcf06eb6-39bd-46b1-b32e-fba76a4020a3.png) | ![Screenshot from 2022-05-10 19-21-10](https://user-images.githubusercontent.com/3187531/167744943-3f0864a7-938d-4bf4-bbbd-315ed3d8045a.png) |
